main/src/ca/uwaterloo/flix/api/lsp/provider/completion/WithHandlerCompleter.scala

```diff
@@ -0,0 +1,25 @@
+package ca.uwaterloo.flix.api.lsp.provider.completion
+
+import ca.uwaterloo.flix.api.Flix
+import ca.uwaterloo.flix.api.lsp.{Index, TextEdit}
+import ca.uwaterloo.flix.language.ast.TypedAst
+
+object WithHandlerCompleter {
+  def getCompletions(context: CompletionContext)(implicit flix: Flix, index: Index, root: TypedAst.Root): Iterable[Completion] = {
+    if (context.previousWord != "with") {
```

Comment:
Ideally this would not be necessary. Do we sometime get the wrong syntactic context or what?

Reply:
I just saw it in some other instances and thought I'd be consistent

Reply:
OK; we are moving away from that "old style". Ideally the syntactic context should be enough.